.changeset/quick-dolphins-roll.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hopper-ui/components": patch
+---
+
+Adjusted the ghost secondary text and icon color
```

Comment:
don't think we need a changeset for the components, just the docs.
@franckgaudin , am I wrong?
also, you're missing the one for docs :P

Reply:
This one should not be in the PR

Reply:
Making a changeset for a component generates a new deployment on NPM. If you want to group several changes in one release, you won't do a changeset at the moment, but if you want to deploy them to get quick feedback, you can.